libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -2483,6 +2535,10 @@ const char *AP_Mission::Mission_Command::type() const
         return "Scripting";
     case MAV_CMD_DO_JUMP:
         return "Jump";
+    case MAV_CMD_DO_JUMP_TAG:
+        return "Jump to Tag";
```

Comment:
Do these get emitted?

Reply:
I don't know. They were added for completeness. If do jump is there, this one should too.

Reply:
yes, they get emitted. In my other 'more featured' branch I added some GCS msgs if a jump fails